distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -92,6 +92,9 @@ solr.username=admin
 # Comma-separated list of metacard types that do not support optimistic updates.
 solr.commit.nrt.metacardTypes=workspace,metacard.query,metacard.list,query-template,attribute-group,resource-note
 
+# Comma-separated list of metacard fields that should never show up as search result highlights
+solr.highlight.blacklist=metadata,metacard_source,metacard-tags
```

Comment:
:exclamation: `metacard_source` is a Solr provider internal field along with `metacard_type_name_txt`, `metacard_type_obj`, and probably others I am forgetting.  The provider should be removing these non-metacard fields from the highlights and should not depend on configuration.  This is leaking an implementation detail that should stay hidden in the provider.

Reply:
Part of this was a holdover before I added the call to `DynamicSchemaResolver.isPrivateField()`.  The remaining ones here after that change (minus metacard-tags) were ones that were not listed as private fields.  Per my other response to your other comment, I can add them to the private list in the resolver or create a new list if that makes more sense.

Reply:
`metacard_source` is still a problem since the Solr provider does not know about that field and only the result cache knows about that field.  It still seems safer to use the returned metacard type definition to filter our highlights than have to maintain this array list which will not know about anything that composes the Solr provider and adds its own private fields like the cache.

https://github.com/codice/ddf/blob/55ce87a/catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java#L113-L118

Reply:
I left the blacklist capability in but removed these types of fields from the list as I now use the metacard type to validate which attributes should have highlighting applied.